sandbox/libs/authn/src/main/java/org/opensearch/authn/jwt/JwtVendor.java

```diff
@@ -79,4 +89,106 @@ public static String createJwt(Map<String, String> claims) {
 
         return encodedJwt;
     }
+
+    public static String createEarlyJwt(Map<String, String> claims) {
```

Comment:
Can you add a brief javadoc here explaining what/why of this method?

Also are these methods (this and the ones below) going to be used outside of test cases? If not we can create a JWTVendorTestUtils class that contains all these test related methods

Reply:
I had put these in here since I figured that a 'vendor' would be the appropriate place to add any additional Jwt creation methods. I can definitely make a different class for the testing methods if that is what you think is best.